sklearn/metrics/pairwise.py

```diff
@@ -1446,12 +1446,12 @@ def _precompute_metric_params(X, Y, metric=None, **kwds):
     """
     if metric == "seuclidean" and 'V' not in kwds:
         if X is Y:
-            V = np.var(X, axis=0, ddof=1)
+            V = np.var(X, axis=0, ddof=1, dtype=np.float64)
```

Comment:
We can do the following to check for version:

```py
        dtype=np.float64 if sp_version < parse_version('1.5') else None
        if X is Y:
            ...
```
And a comment explaining why this needs to be done.

(Where `sp_version` is in `sklearn.utils.fixes`.)

Reply:
Thanks for that.  This is fixed in 3acc8fe